### PR TITLE
returns complete info in disk.getinfo command

### PR DESCRIPTION
**Pull request by zaibon**
relates to https://github.com/g8os/core0/issues/91

I tried to implement test for this:
```go
package builtin

import (
	"testing"

	"github.com/g8os/core0/base/pm"
)

func TestDiskInfo(t *testing.T) {
	pm.InitProcessManager(10)
	mgr := diskMgr{}
	_, err := mgr.lsblk("nvme0n1p1")
	if err != nil {
		t.Errorf(err.Error())
		t.FailNow()
	}
}
```

But the runner blocks on `Wait` forever and the test never finishes. do I need to initialize something else ?

### CODE REVIEW

**Comment by muhamadazmy** [comment]
You inited the process manager, but u never did a 'Run()' so it's intialized but not processing any commands.

File: core0/builtin/disk.go [hunk 1]
```diff
@@ -62,6 +112,45 @@ func (d *diskMgr) readUInt64(p string) (uint64, error) {
...
+		return nil, fmt.Errorf("failed to set run lsblk command: %v", cmdResult.Streams)
+	}
+
+	stdout := ""
+	if len(cmdResult.Streams) > 1 {
+		stdout = cmdResult.Streams[0]
+	}
+
+	cmdOutput := struct {
+		BlockDevices []*lsblkResult `json:"blockdevices"`
```
Comment (muhamadazmy) [inline core0/builtin/disk.go]: No need to define it as `[]*lsblkResult` a `[]lsblkResult` is cleaner and same size of allocation.

**Comment by zaibon** [comment]
@muhamadazmy check the lsblk command, it calles `runner.Wait()` and from what I understood this calls Run unternally,no ?

**Comment by muhamadazmy** [comment]
@zaibon I mean your test, u need to do `pm.GetManager().Run()` to actually start the process manager itself. 
Then u can call pm.GetManager().RunCmd()

**Comment by muhamadazmy** [comment]
https://github.com/g8os/core0/blob/0.12.0/core0/main.go#L87

**Comment by zaibon** [comment]
understood, thx !

**Pull request merged**